PGIP/Shared.hs

```diff
@@ -1,4 +1,12 @@
 {-# LANGUAGE CPP, DoAndIfThenElse #-}
+{- |
+Module      : ./PGIP/Shared.hs
+Description : Provides various Hets resources.
```

Comment:
this description is too unspecific. Better: "Provides resources for caching requests to the RESTful interface"